common/predictive-text/unit_tests/headless/edit-distance/classical-calculation.js

```diff
@@ -456,4 +456,78 @@ describe('Classical Damerau-Levenshtein edit-distance calculation', function() {
       assert.equal(trimmedBuffer.getFinalCost(), 2)
     });
   });
+
+  describe("edit path construction", function() {
+    let op = {
+      s: 'substitute',
+      m: 'match',
+      i: 'insert',
+      d: 'delete',
+      ts: 'transpose-start',
+      te: 'transpose-end',
+      ti: 'transpose-insert',
+      td: 'transpose-delete'
+    }
+
+    it("'accomodate' -> 'accommodate'", function() {
+      let buffer = compute("accomodate", "accommodate", "InputThenMatch");
+  
+      let editSequence = [
+      //  a     c     c     o     m
+        op.m, op.m, op.m, op.m, op.m,
+      // *m    o      d     a     t    e
```

Comment:
Do you mark +m for `op.i`?

Reply:
... yeah, kinda realized it after the PR went up... and after I'd split branch into two versions (as the changes are _also_ in #3549).  I'm being a bit lazy, but I could adjust it to use the same `+m` syntax.

Reply:
So, as noted:

```suggestion
      // +m    o      d     a     t    e
```